aiohttp/web_protocol.py

```diff
@@ -212,6 +212,9 @@ def __init__(
     ):
         super().__init__(loop)
 
+        # _request_count is the number of request processed by the handler
+        # A new handler is created for each connection, so it is the number
+        # of requests processed by the connection.
```

Comment:
I find this slightly confusing. I'd probably avoid the word handler here as I'm thinking of the endpoint handler (i.e. the function registered on a route) which is presumably not what you mean.

Reply:
I simplified the comment. Hopefully it makes more sense now